### ISSUE TITLE
Can't install since Pebble servers shut down

### ISSUE THREAD

**Issue by ekeih** [markusressel/Watchface-No.-2#1]
I configured the Rebble store instead of the Pebble store. But I am not able to install your awesome watchface with it. The Android "open with" dialog does not offer "Pebble" as an option.
Do you have any idea what I could do?

**Comment by markusressel** [comment]
Hey @ekeih,
thanks for opening this issue. I have the exact same problem (coincidence?) you described.
**Unfortunately I don't have a fix for this (yet).**
I hope when Rebble is ready to host their own version of the cloud developing platform Pebble used to have (CloudPebble) it will be possible to install ` .pbw`  files from there but at the moment there is no way that I am aware of.

**Comment by markusressel** [comment]
I recently found a way to install `.pwb` files using [Gadgetbridge](https://github.com/Freeyourgadget/Gadgetbridge). I was able to connect my pebble to the gadgetbridge app without unpairing it from the pebble/rebble app (I had to disconnect it in the pebble. app though). I learned from the gadgetbridge docs that pebble used it's cloud infrastructure to remember what apps you installed on your watch which is not supported by gadgetbridge so will not be able to see the watchface in the pebble app if you install it via gadgetbridge. The gadgetbridge app also does not request the internet permission. Therefore weather forecast will not work as this requires an online api.
